src/scss/components/_list.scss

```diff
@@ -100,4 +109,97 @@
       padding-right: inherit;
     }
   }
+
+  &__title {
+    color: get-color(rastaGray);
+    display: block;
+    font-size: 12px;
+    font-weight: get-weight(bold);
+    text-align: left;
+    text-transform: uppercase;
+  }
+
+  &--on-column {
+    &[class*='col-'] {
+      display: inline-block;
+      margin-left: 0;
+
+      + & {
+        margin-left: auto;
+        margin-right: 0;
+      }
+
+      .list__item {
+        width: 100%;
+      }
+    }
+  }
+
+  &--social {
+    display: flex;
+    flex-grow: 1;
+
+    .list__item {
+      border-radius: 50%;
+      height: 48px;
+      margin-left: 6%;
+      text-indent: -9999px;
+      width: 48px;
+
+      a {
+        padding: 0;
+        width: 100%;
+      }
+
+      img {
+        display: block;
+        width: 100%;
```

Comment:
`width: 100%` is not a good approach to images. Use the appropriated image sizes decreases our page load and improves the mobile load of our page. We have other images with this approach and we need to review this choices.